src/features/users.ts

```diff
@@ -0,0 +1,29 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { User } from '../types/User';
+import { getUsers } from '../api/users';
+
+type UserState = {
+  users: User[];
+};
+
+const initialState: UserState = {
+  users: [],
+};
+
+export const fetchUsers = createAsyncThunk('users/fetch', () => {
+  return getUsers();
+});
```

Comment:
Consider adding handling for the pending and rejected states of the `fetchUsers` action in the `extraReducers` to provide better feedback in the UI.